code/modules/wiremod/shell/brain_computer_interface.dm

```diff
@@ -459,9 +450,9 @@
 
 	var/mob/living/carbon/carbon_occupant = occupant
 	if (istype(occupant))
-		var/obj/item/organ/internal/cyberimp/bci/existing_bci_organ = carbon_occupant.getorgan(/obj/item/organ/internal/cyberimp/bci)
-		if (isnull(existing_bci_organ) && isnull(bci_to_implant?.resolve()))
-			say("No brain-computer interface inserted, and occupant does not have one. Insert a BCI to implant one.")
+		var/obj/item/organ/internal/cyberimp/bci/bci_organ = carbon_occupant.getorgan(/obj/item/organ/internal/cyberimp/bci)
+		if (!bci_organ && !bci_to_implant)
+			say("No brain-computer interface inserted and the occupant doesn't have one. Insert a BCI to implant one.")
```

Comment:
Why?

Reply:
Guh, preference. Will revert the text.

Reply:
I do want it reverted I think it's needless

Reply:
Will do.